### DISCUSSION TITLE
The Barnacle

### DISCUSSION THREAD

**Discussion by Veritius** [FakeFishGames/Barotrauma#4684] [Ideas and suggestions]
Copy from issue #4595 which was created before Discussions was setup.

**Barnacle**
Attaches to hull of ship and begins to rapidly damage the hull in a slowly expanding region starting at the barnacle itself.
A glowing red "core" in its fleshy mass must be destroyed to kill it, but cannot be harmed effectively from inside the sub, requiring someone to go outside and stab/shoot it. Completely immobile itself and moves with the sub. Method of attachment is drifting into a submarine. Players will become aware of attachment by an impact similar to a Moloch or Hammerhead but without the damage.

**Comment by Rokvach** [comment]
I've had the same idea back in May last year, but currently its something that can't yet be implemented as we would need to teach the AI to handle maintenance outside of the submarine, otherwise it would be a bit too difficult to handle in singleplayer. But there could also be a outpost service to clean it out from the sub, but that would mean you would need to reach a outpost so the effects of the barnacle can't be anything drastic then. 

> I've been thinking of a new maintenance task, now that repairs can all be done from the safety of the submarine there isn't much maintenance the crew would need to do outside of the submarine.
> Barnacles, especially some massive whale barnacles could be a good addition to the overall maintenance route, where if you scrape the submarine a bit too close to the cave walls or leave the submarine stationary, these barnacles could attach to the submarine hull, causing the submarine to travel a bit slower. Barnacles could also be beneficial to the crew if they tolerate the slowness they cause, benefit would come from added structural integrity to the areas where the barnacles are attached to.
> They should be able to be removed from the submarine walls by plasma cutting them outside the submarine, should also give some minerals.
> Barnacles could also be limited to a specific biome and the crew could possibly upgrade their structural walls to something which eliminates these barnacles from sticking to it. Barnacles could also be accumulative, so having some of them attached shouldn't affect much to the submarine but untreated they would just get worse.
> 
> They could create a rather interesting balance, where players might intentionally try and gather the barnacles in the weak spots of the submarine where they would then need to nurture them and knock off the excess so the submarine wouldn't be slowed down too much, the barnacles would however be knocked lose by creature attacks so replenishing would be necessary from time to time.

**Reply by Veritius** [reply]
Not the same, but similar

**Reply by TeaandCrumpets** [reply]
I'd like to see a barnacle which attaches to the sub and is harmless unless attacked, then it explodes. The only way to remove them would be with a crowbar from the outside.

**Reply by Saevon** [reply]
I would love these kinds of creatures (like the spores) to have various forms, each with a creature-type gimmick

Barnacles attach to hull from the sea-floor / Sea-Ceiling, but also from certain "barnacle covered creatures" like maybe passive whales. They slowly spread from where-ever they're "planted"

 * Explody barnacles
 * Hull chewy barnacles (that slowly replace Shell/Wall HP with themselves until its eaten away, then they physically flow in through the breach… re attaching once they stop moving)
 * Mineral barnacles, useful if cut open, or allowed to spread (kill them for minerals). But fall off easily from any nearby attacks / sonar noise (like engines or reactor).

Spores spread from contaminated waters (e.g from certain seas, or near certain plantlife) and grow a large plant-mass while mature.

 * Electrospores eat electricity and can controls electronics like hatches and pumps
 * Hermit spores spread from container to containner. Three phases: Young wait to mature (invisible), Mature spread spores (visible stains), Old create massive trees, locking the container (and space) for a time, then die and create new spores inside it.
   * Thus crew spread Mature spores from container to container! better get hope you know all the contraband containers that people go to… (or force people to go to decontamination often)… Spawn as spores on a character
 * Gas Spores eat oxygen, spreading while a room has oxygen. Breaking vents and getting a growth spurt (if they reach them), opening infested doors to new oxygen (or to floods / danger) or closing them if attacked… Spawn on a vent or Oxygen generator.

**Comment by RadCatish** [comment]
I had a similar idea, but thought that maybe it could "grow", not damaging the hull until it's full size, and dropping materials if removed from the hull at medium size
